applications/registration-react/src/pages/dataset-registration-page/dataset-registration-page.scss

```diff
@@ -19,12 +19,30 @@
 .form-group {
   margin-bottom: 4.8rem;
   .fdk-form-label {
+    display: block;
+    position: relative;
     font-weight: 500;
     margin-bottom: 0;
-  }
 
-  .fdk-form-field {
+    .language-indicator {
+      position: absolute;
+      top: 10px;
```

Comment:
Vurdere om du heller skal bruke 'em' på margins og paddings for å være mer responsiv. Hvordan ser det ut med 'px' her hvis du zoomer til 200%? 'em' er relativ til parent font-size.